osutil/group.go

```diff
@@ -156,3 +184,28 @@ func FindGid(group string) (uint64, error) {
 	//return strconv.ParseUint(group.Gid, 10, 64)
 	return strconv.ParseUint(group, 10, 64)
 }
+
+// FindGid returns the identifier of the given UNIX group name.
+func FindGroup(gid uint64) (string, error) {
```

Comment:
Can we write tests for this? A simple smoke test based on the local user or something?

Reply:
Ha, the reason this started was that there's just nothing sane about groups across systems. 

Still +1 on a test.

Reply:
Added some tests to cover find group ID/name and group owning a particular file.